app/models/event.rb

```diff
@@ -6,12 +6,50 @@
 #  name             :string
 #  description      :string
 #  max_participants :integer
+#  date_ranges      :Collection
 #  active           :boolean
 #  created_at       :datetime         not null
 #  updated_at       :datetime         not null
 #
+
 class Event < ActiveRecord::Base
+  UNREASONABLY_LONG_DATE_SPAN = 300
```

Comment:
As a configuration parameter, I think this doesn't really belong into the model class